### PR TITLE
Logging to find att segmentation fault

### PR DESCRIPTION
**Pull request by joakimmoller**
* replace prints with logging
* building oops consistently
* remove old binary (libodin.a)
* remove installation requirements from setup.py (fullfilled by requirement.in)

Possible outcome: qa, qe are zero for attitude data?

Check for latest available att data in the database:
```
odin=> select stw, qt, qa, qe from attitude_level0 order by stw desc limit 10;
     stw     |                     qt                      |                     qa                      |    qe
-------------+---------------------------------------------+---------------------------------------------+----------
 14274920153 | {0.69861,-0.6022504,0.2692111,-0.2770629}   | {0.7007634,-0.600115,0.2739254,-0.2715833}  | {10,4,2}
 14274920136 | {0.6987708,-0.6020943,0.2695625,-0.2766549} | {0.700886,-0.5999902,0.2741905,-0.2712751}  | {10,4,2}
 14274920119 | {0.6989314,-0.601938,0.2699137,-0.2762467}  | {0.7010028,-0.5998727,0.2744495,-0.2709712} | {10,4,2}
 14274920102 | {0.6990917,-0.6017814,0.2702649,-0.2758384} | {0.7011179,-0.5997551,0.2747102,-0.2706692} | {10,4,2}
 14274920085 | {0.6992518,-0.6016246,0.270616,-0.2754301}  | {0.7012353,-0.5996356,0.2749725,-0.2703633} | {10,4,2}
 14274920068 | {0.6994116,-0.6014677,0.270967,-0.2750216}  | {0.70135,-0.5995212,0.2752302,-0.2700573}   | {10,4,2}
 14274920051 | {0.6995713,-0.6013105,0.2713179,-0.2746131} | {0.7014673,-0.5994032,0.2754887,-0.2697507} | {10,4,2}
 14274920034 | {0.6997307,-0.6011531,0.2716687,-0.2742044} | {0.7015824,-0.5992867,0.2757485,-0.2694447} | {10,4,2}
 14274920017 | {0.6998898,-0.6009955,0.2720195,-0.2737956} | {0.7016986,-0.5991686,0.2760105,-0.2691362} | {10,4,2}
 14274920000 | {0.7000487,-0.6008377,0.2723701,-0.2733868} | {0.7018149,-0.599049,0.276272,-0.2688308}   | {10,4,2}
(10 rows)
```

STW's around the first scan for a file that gets segmentation fault:

```
odin=> select stw, qt, qa, qe from attitude_level0 where stw between 14233165866 and 14233408208 order by stw desc limit 10;
     stw     |    qt     |                     qa                      |    qe
-------------+-----------+---------------------------------------------+----------
 14233408193 | {0,0,0,0} | {0.701077,-0.3850955,0.5360506,-0.269893}   | {20,8,3}
 14233408176 | {0,0,0,0} | {0.7012752,-0.384705,0.5363362,-0.2693672}  | {20,8,3}
 14233408159 | {0,0,0,0} | {0.701472,-0.3843152,0.53662,-0.2688456}    | {20,8,3}
 14233408142 | {0,0,0,0} | {0.701674,-0.3839163,0.5369046,-0.2683195}  | {20,8,3}
 14233408125 | {0,0,0,0} | {0.7018717,-0.3835146,0.5371895,-0.2678061} | {20,8,3}
 14233408108 | {0,0,0,0} | {0.7020702,-0.3831177,0.5374722,-0.2672863} | {20,8,3}
 14233408091 | {0,0,0,0} | {0.7022687,-0.3827206,0.5377554,-0.2667633} | {20,8,3}
 14233408074 | {0,0,0,0} | {0.7024655,-0.3823193,0.53804,-0.2662464}   | {20,8,3}
 14233408057 | {0,0,0,0} | {0.7026619,-0.3819234,0.5383217,-0.2657266} | {20,8,3}
 14233408040 | {0,0,0,0} | {0.7028589,-0.3815235,0.5386057,-0.2652039} | {20,8,3}
(10 rows)
```

qt are zero!

### CODE REVIEW

**Comment by joakimmoller** [comment]
A handful rows in attitude_level0 has qt set to zero others have qe and qa set to zero.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,25 +35,20 @@ jobs:
...
-        - name: Run unit tests
+        - name: Run unit tests odincal
           run: |
-            cd odincal/oops
-            tox
\ No newline at end of file
+            cd odincal/odincal
+            tox
+  
+
```
Comment (skymandr) [inline .github/workflows/ci.yml]:
```suggestion

```

File: odincal/Dockerfile [hunk 2]
```diff
@@ -17,12 +29,10 @@ COPY requirements.txt .
...
 
 # Install oops and odincal
-COPY ./oops /oops
-WORKDIR /oops/oops
-RUN gcc -shared -o libfft.so -fPIC ../Library/odinfft.c
+COPY --from=build /oops/dist/oops-1.1.1-cp27-cp27mu-linux_x86_64.whl /
 WORKDIR ${LAMBDA_TASK_ROOT}
 COPY ./odincal /odincal
-RUN PYTHONPATH="${LAMBDA_TASK_ROOT}" pip install /oops --target "${LAMBDA_TASK_ROOT}"
+RUN pip install file:///oops-1.1.1-cp27-cp27mu-linux_x86_64.whl --target "${LAMBDA_TASK_ROOT}"
```
Comment (skymandr) [inline odincal/Dockerfile:35]: ska det verkligen vara tre /?

File: odincal/requirements.in [hunk 3]
```diff
@@ -1,9 +1,9 @@
 attrs
-numpy==1.15.4
+numpy
```
Comment (skymandr) [inline odincal/requirements.in:2]: hur fick du detta att funka?

**Review by skymandr — APPROVED** [review]
LGTM

File: odincal/requirements.in (hunk 3 above)
Comment (joakimmoller) [reply odincal/requirements.in:2]: pip-compile (py2.7 ) väljer snällt de versionerna som passar.

File: odincal/Dockerfile (hunk 2 above)
Comment (joakimmoller) [reply odincal/Dockerfile:35]: ja, schemat är `<protocol>://<host>/<path>` -> file:/// ingen host, root filsystemet

**Pull request merged**